apps/src/lib/levelbuilder/lesson-editor/FindProgrammingExpressionDialog.jsx

```diff
@@ -0,0 +1,217 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+
+import Button from '@cdo/apps/templates/Button';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+
+import LessonEditorDialog from './LessonEditorDialog';
+
+const SearchForm = function(props) {
+  return (
+    <form className="form-search">
+      <input
+        type="text"
+        className="input-large search-query"
+        onChange={props.onSearch}
+        placeholder="search by name"
+      />
+      <select
+        className="input-small search-query"
+        style={{marginLeft: 4}}
+        onChange={props.onFilter}
+      >
+        <option value="">filter</option>
+        {/* TODO: these should probably be sourced from somewhere, rather than hardcoded */}
```

Comment:
[tracked here](https://codedotorg.atlassian.net/browse/PLAT-881)

Reply:
I can pick this up as part of my work. I already have it set up to pass this information to the component